raster/r.proj/main.c

```diff
@@ -253,11 +253,8 @@ int main(int argc, char **argv)
     setname = imapset->answer ? imapset->answer : G_store(G_mapset());
     if (strcmp(inlocation->answer, G_location()) == 0 &&
         (!indbase->answer || strcmp(indbase->answer, G_gisdbase()) == 0))
-#if 0
-	G_fatal_error(_("Input and output locations can not be the same"));
-#else
 	G_warning(_("Input and output locations are the same"));
```

Comment:
r.proj does not make sense within the same location, instead one of the r.resamp.* modules should be used. Please add this to the warning and something like "nothing to do" and exit with exit(EXIT_SUCCESS).